pkg/apis/policy/validation/validation.go

```diff
@@ -40,7 +41,7 @@ import (
 // with any errors.
 func ValidatePodDisruptionBudget(pdb *policy.PodDisruptionBudget) field.ErrorList {
```

Comment:
Actually, this should drop calling `ValidatePodDisruptionBudgetStatus` entirely. This method is only called from `podDisruptionBudgetStrategy#Validate` and `podDisruptionBudgetStrategy#ValidateUpdate`, which disallow making any changes to status. If ValidatePodDisruptionBudgetStatus returned an error here, the API caller to those endpoints would have no way to correct it.